main.go

```diff
@@ -1,112 +1,115 @@
 //go:generate bash ./scripts/controller-gen-generate.sh
+
 package main
 
 import (
 	"context"
-	"fmt"
+	"flag"
 	"net/http"
 	"os"
 	"time"
 
-	"github.com/rancher/gitjob/pkg/controller"
-	"github.com/rancher/gitjob/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
 	"github.com/rancher/gitjob/pkg/webhook"
-	"github.com/rancher/wrangler/v2/pkg/leader"
-	"github.com/rancher/wrangler/v2/pkg/ratelimit"
-	"github.com/rancher/wrangler/v2/pkg/resolvehome"
-	"github.com/rancher/wrangler/v2/pkg/signals"
-	"github.com/sirupsen/logrus"
-	"github.com/urfave/cli"
-	"k8s.io/apimachinery/pkg/util/runtime"
-	"k8s.io/client-go/tools/clientcmd"
+
+	gitjobv1 "github.com/rancher/gitjob/pkg/apis/gitjob.cattle.io/v1"
+	"github.com/rancher/gitjob/pkg/controller"
+	"github.com/rancher/gitjob/pkg/git/poll"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
 )
 
 var (
-	Version   = "v0.0.0-dev"
-	GitCommit = "HEAD"
```

Comment:
These are not used [anymore](https://github.com/rancher/gitjob/pull/368/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L32) since urfave/cli is not used for the cli arguments anymore

Reply:
removed in the script https://github.com/rancher/gitjob/pull/368/commits/f572252f7866ea550c183541a87797f5881d2ba3